_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -3277,6 +3277,12 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
         extOpt2->MBBRC = MFX_CODINGOPTION_OFF;
     }
 
+    if (extOpt3->WinBRCSize && !hwCaps.ddi_caps.FrameSizeToleranceSupport)
+    {
+        unsupported = true;
```

Comment:
It looks like we need to modify this line: https://github.com/Intel-Media-SDK/MediaSDK/blob/089296a90c83ed54e95b20597437a6839406e10b/_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp#L234

Reply:
Done.